diff --git a/libacars/CMakeLists.txt b/libacars/CMakeLists.txt
index 01d2c71..ee1d958 100644
--- a/libacars/CMakeLists.txt
+++ b/libacars/CMakeLists.txt
@@ -214,6 +214,12 @@ install(TARGETS acars
 	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
 	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
 )
+install(TARGETS acars_static
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+)
+
 install(FILES
 	acars.h
 	adsc.h
